modules/manifold/package.json

```diff
@@ -1,23 +1,20 @@
 {
   "name": "@mlvis/manifold",
-  "version": "1.0.0",
+  "version": "1.0.0-alpha.12",
   "publishConfig": {
     "access": "public"
   },
   "description": "Model-Agnostic Visual Debugging Tool for Machine Learning",
   "license": "Apache-2.0",
   "author": "Firenze11 <lezhi.li@uber.com>",
-  "main": "dist/index.js",
+  "main": "index.js",
```

Comment:
For packages that allow for `import {something} from '@mlvis/manifold/some-subdirectory';`, their entry point has to be `index.js` instead of `dist/esm/index.js`

Reply:
Hello, ever consider to config ocular to only build one bundle into dist/ and that being the esm/. That way you enable tree-shaking as well as the folder export, both of them have merits on reducing bundle size, not mentioning the saved build time by only building one bundle.